tests/unit/test__pandas_helpers.py

```diff
@@ -734,6 +734,35 @@ def test_list_columns_and_indexes_with_named_index_same_as_column_name(
     assert columns_and_indexes == expected
 
 
+@pytest.mark.skipif(pandas is None, reason="Requires `pandas`")
+def test_dataframe_to_json_generator(module_under_test):
+    utcnow = datetime.datetime.utcnow()
+    df_data = collections.OrderedDict(
+        [
+            ("a_series", [pandas.NA, 2, 3, 4]),
+            ("b_series", [0.1, float("NaN"), 0.3, 0.4]),
+            ("c_series", ["a", "b", pandas.NA, "d"]),
```

Comment:
If I understood correctly, `pandas.NA` was introduced in pandas 1.0?

In `setup.py` pandas is currently pinned to `>=0.23.0`, meaning that this test will fail. We should modify the `skipif` condition to also skip if too old a version is installed.

Reply:
Very good point. I have copied the `PANDAS_..._VERSION` syntax from `test_client` and updated the syntax as you suggested.

`pytest.importorskip` would condense the logic a bit but I can see you use `skipIf` everywhere else so consistency is perhaps best here.

```python
def test_dataframe_to_json_generator(module_under_test):
    pytest.importorskip(
        "pandas",
        minversion="1.0.0",
        reason="Requires `pandas version >= 1.0.0` which introduces pandas.NA"
    )
```

Reply:
Currently both are used. I actually like `impororskip` more, but just copy/pasted the first existing example I found. :slightly_smiling_face: 

We should make conditionally skipping the tests internally consistent, but that's a thing for a separate PR.